src/scripts/react/pages/main.js

```diff
@@ -1,3 +1,5 @@
+import "isomorphic-fetch";
```

Comment:
For what?

Reply:
We are making requests to API in charts components using fetch. It's polyfill for fetch

Reply:
Ahh, ok. Maybe we can use separated file for all polyfills like `polyfill.js`?